releasenotes/notes/expose-clc-runner-stats-dad4462c9c661f3a.yaml

```
@@ -0,0 +1,7 @@
+---
+features:
+  - |
+    The Agent can now expose runner stats via the CLC Runner API Server, a remotely-accessible authenticated API server.
+    The Cluster Agent can use these stats to optimize dispatching cluster level checks.
+    The CLC Runner API Server is disabled by default, must be enabled via the Agent config, the cluster agent must enabled since it's the only client of the server.
```

Looks more straightforward to rephrase to:

```suggestion
    The CLC Runner API Server is disabled by default, it must be enabled in the Agent configuration.
```

Since you already state above that the DCA can use these stats.

---

`the cluster agent must enabled since it's the only client of the server.` is because we start the server only if the cluster agent is enabled :
```
	// start clc runner server
	// only start when the cluster agent is enabled and a cluster check runner host is enabled
	if config.Datadog.GetBool("cluster_agent.enabled") && config.Datadog.GetBool("clc_runner_enabled") {
		if err = clcrunnerapi.StartCLCRunnerServer(); err != nil {
			return log.Errorf("Error while starting clc runner api server, exiting: %v", err)
		}
	}
```

---

also wanted to satisfy what has been suggested here https://github.com/DataDog/datadog-agent/pull/3931#discussion_r312860514 `whether the API is enabled by default or not, and under which conditions it's enabled by default`

---

allright 👍